book/content/make/make.md

```diff
@@ -694,6 +705,102 @@ The published Makefile in the repository does not contain the paper, but this
 collaboration as only a single repository needed to be shared that contains 
 the code, the results, and the manuscript.
 
+## Including numerical results and tables
+
+At this point you may be thinking "That's so cool that I can now include figures
+into my manuscripts! But how exactly does this work for numerical results?"
+
+The reproducible paper linked above shows one way of doing it:
+After the results are computed, they are written out in the form of a LaTeX
+table.
+Here is how one of these tables looks like right after it was computed:
+
+```
+\begin{tabular}{lrrr|rrrr}
+Property & HypoParsr & Sniffer & Suitability & Pattern & Type & No Tie & Full\\
+\hline
+Delimiter & 87.48 & 86.82 & 65.41 & 92.61 & 88.33 & 91.38 & \textbf{94.92}\\
+Quotechar & 82.90 & 92.36 & 44.60 & 95.23 & 90.10 & 93.80 & \textbf{97.36}\\
+Escapechar & 87.96 & 94.37 & 74.85 & 97.95 & 96.26 & 95.44 & \textbf{99.25}\\
+Overall & 80.60 & 85.45 & 38.19 & 90.99 & 83.61 & 90.61 & \textbf{93.75}\\
+\hline
+```
+
+To include this table into your manuscript, you can use LaTeX's `\input{}`
+function. If the file with the table is called `mytable.tex`, this command
+can insert it into your manuscript:
+
+```
+\begin{table}
+\input{mytable}
+\end{table}
+```
+
+An alternative is to make use of variables:
```

Comment:
This colon should probably be a period too :)

Incidentally, the CSV project [uses this strategy](https://github.com/alan-turing-institute/CSV_Wrangling/tree/master/results/test/analysis/constants) for all the single-number results in the text (as you may be aware)